src/shape/bounding-box.js

```diff
@@ -298,6 +298,51 @@ pc.extend(pc, function () {
             }
 
             this.setMinMax(min, max);
+        },
+
+        /**
+         * @function
+         * @name pc.BoundingBox#intersectsBoundingSphere
+         * @description Test if a Bounding Sphere is inside a AABB.
```

Comment:
Intersection is not necessary when sphere is inside of an AABB, it might be just touching each other, which is still "intersecting".

So perhaps reference should be slightly rephrased.

Additionally, it would be useful to know if it is just colliding, or fully contained. This could be done by returning a number.
0 - no intersection
1 - intersects
2 - contains

This is still fully compatible with `if` conditioning, as `1 == true && 2 == true`. And allows developer to have a bit more information about intersection specifics.